### PR TITLE
1309 refactor bootstrap element progress

### PR DESCRIPTION
**Pull request by ArKhmeln**

### CODE REVIEW

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/progress/ProgressBackgroundTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultiplebarsProgressTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4505869&bid=15186778)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultiplebarsProgressTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/progress/ProgressBackgroundTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4505869&bid=15186778)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultiplebarsProgressTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/progress/ProgressBackgroundTests.java  1
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/MultiplebarsProgress.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4505869&bid=15186778)

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java [hunk 1]
```diff
@@ -285,6 +286,12 @@
     public static Progress progressBackgroundYellow;
     @UI("#progress-backgrounds-red")
     public static Progress progressBackgroundRed;
+    @UI("#progress-multiple-bars")
+    public static MultiplebarsProgress multiplebarsProgress;
+    @UI("#navbar-containers-centred")
+    public static Navbar navbarCentredContainer;
+    @UI("#navbar-containers-expanded")
+    public static Navbar navbarExpandedContainer;
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: Why was this navbars added during this task?

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/MultiplebarsProgress.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultiplebarsProgressTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/progress/ProgressBackgroundTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4505869&bid=15186778)

**Pull request merged**